### PR TITLE
fix: entity commondimension aligned with telemetry sdk

### PR DESCRIPTION
**Pull request by rogercoll**
Align common data with telemetry API: https://github.com/newrelic/newrelic-telemetry-sdk-go/blob/master/telemetry/metrics.go#L241

Infra agent: https://github.com/newrelic/infrastructure-agent/blob/30d0e84f3526ab0e1398d7908cc6e73193a21109/pkg/backend/telemetryapi/metrics_test.go#L52

### CODE REVIEW

File: integration/entity.go [hunk 1]
```diff
@@ -60,7 +70,25 @@ func (e *Entity) AddCommonDimension(key string, value string) {
...
+	t := timestamp.Unix()
+	e.CommonDimensions.Timestamp = &t
+}
+
+// AddCommonInterval adds a new common timestamp to the entity from duration in nanoseconds.
+func (e *Entity) AddCommonInterval(timestamp time.Duration) {
+	e.lock.Lock()
+	defer e.lock.Unlock()
+
+	t := timestamp.Milliseconds()
```
Comment (rubenruizdegauna) [inline integration/entity.go:90]: comment above says nanoseconds. Is this ok? ☝️

File: integration/entity_test.go [hunk 2]
```diff
@@ -217,26 +217,35 @@ func Test_Entity_EntitiesWithSameMetadataAreSameAs(t *testing.T) {
 
 func TestEntity_AddCommonDimension(t *testing.T) {
 	tests := []struct {
-		name      string
-		commons   metric.Dimensions
-		expeceted *Entity
+		name     string
+		commons  metric.Dimensions
+		expected *Entity
```
Comment (rubenruizdegauna) [inline integration/entity_test.go:222]: what's wrong with `expeceted`??? :trollface:

**Review by rubenruizdegauna — APPROVED** [review]
LGTM! just a comment to clarify interval value

File: integration/entity_test.go (hunk 2 above)
Comment (rogercoll) [reply integration/entity_test.go:222]: It is not familiar to me, YET 😂

File: integration/entity.go (hunk 1 above)
Comment (rogercoll) [reply integration/entity.go:90]: The function parameter is in nanoseconds, the resulting interval in milliseconds.

New commit changed the comment as it was misleading. 

Thanks!

Comment (rubenruizdegauna) [reply integration/entity.go:90]: got it. Thanks!!

**Review by rubenruizdegauna — APPROVED** [review]
LGTM!

**Pull request merged**